File: src/codes/Users_Manual/silencer/silencer_fixed_completion_steps.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
let steps_intensity = 10;
let steps_phase = 10;
let config = Silencer::from_completion_steps(steps_intensity, steps_phase);
// or
let time_intensity = std::time::Duration::from_micros(250);
let time_phase = std::time::Duration::from_micros(250);
let config = Silencer::from_completion_time(time_intensity, time_phase);
# }

File: src/codes/Users_Manual/silencer/silencer_fixed_completion_steps.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~int main() {
//~{
const auto steps_intensity = 10;
const auto steps_phase = 10;
const auto config =
    autd3::Silencer::from_completion_steps(steps_intensity, steps_phase);
//~}
// or
//~{
const auto time_intensity = std::chrono::microseconds(250);
const auto time_phase = std::chrono::microseconds(250);
const auto config = autd3::Silencer::from_completion_time(time_intensity, time_phase);
//~}
//~return 0; }